### DISCUSSION TITLE
Can service creation/deletion notifications be hidden based on the service/process name?

### DISCUSSION THREAD

**Discussion by korzynski** [winsiderss/systeminformer#988] [Questions / Answers] [answered]
Is it possible to specify a list of services that should not generate service creation/deletion notifications in process hacker? I like the notification feature, but some processes, such as PDQ Deploy, often create and delete services every few minutes and this can be distracting. It would be helpful to be able to "hide" the notification for certain services either by name or by parent process.

**Comment by dmex** [comment, marked as answer]
Yes, see the `filtering notifications` sections on the options window.
